examples/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "winterfell"
+name = "examples"
```

Comment:
I would prefer a better name here, i.e., applications or something similar (I prefer gadgets, but unfortunately they are not necessarily reusable, so let's pick something else).

Reply:
I thing `gadgets` makes sense to me. Will rename in another PR.